packages/react/src/components/Dropdown/Dropdown.stories.js

```diff
@@ -1,27 +1,27 @@
 /**
  * Copyright IBM Corp. 2016, 2018
- *
+ *dropdow.stor
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
 import React from 'react';
-import { default as Dropdown, DropdownSkeleton } from '../';
-import { Layer } from '../../Layer';
-import mdx from '../Dropdown.mdx';
+import { default as Dropdown, DropdownSkeleton } from './';
+import { Layer } from '../Layer';
+import mdx from './Dropdown.mdx';
 
 export default {
   title: 'Components/Dropdown',
   component: Dropdown,
   subcomponents: {
     DropdownSkeleton,
   },
-  argTypes: {
-    size: {
-      options: ['sm', 'md', 'lg'],
-      control: { type: 'select' },
-    },
-  },
+  // argTypes: {
+  //   size: {
+  //     options: ['sm', 'md', 'lg'],
+  //     control: { type: 'select' },
+  //   },
+  // },
```

Comment:
Can this be deleted? Looks like `size` works properly with it's definition in the rest of the arg types